src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -70,19 +71,19 @@
  * @author Oliver Burn
  * @author Andrei Selkin
  */
-public class ParameterNameCheck
-    extends AbstractNameCheck {
+public class ParameterNameCheck extends AbstractNameCheck {
 
     /**
      * Allows to skip methods with Override annotation from validation.
      */
     private boolean ignoreOverridden;
 
-    /** The visibility scope where methods are checked. */
-    private Scope scope = Scope.ANONINNER;
+    /** Access modifiers of methods which should not be checked. */
+    private AccessModifier[] excludeAccessModifiers;
 
-    /** The visibility scope where methods shouldn't be checked. */
-    private Scope excludeScope;
+    /** Access modifiers of methods which should be checked. */
+    private AccessModifier[] accessModifiers =
+        Stream.of(AccessModifier.PUBLIC).toArray(AccessModifier[]::new);
```

Comment:
Oirignal check looked at all modifers, not just `public`. I think we should continue this.
If we don't specify all, I think more examples in xdoc need to be changed/added.